.travis.yml

```diff
@@ -12,5 +12,6 @@ before_script:
   - docker-compose -f docker-compose.yml -f docker-compose.ci.yml up --build -d
 
 script:
-  - docker exec -it cvat /bin/bash -c 'tests/node_modules/.bin/karma start tests/karma.conf.js'
+  - docker exec -it cvat /bin/bash -c 'python3 manage.py test --settings cvat.settings.testing cvat/apps/engine && tests/node_modules/.bin/karma start tests/karma.conf.js'
```

Comment:
@EvgenyShashkin , do we need `--settings cvat.settings.testing` here?

Reply:
@nmanovic , yes, because file testing.py contains functions for working with fakeredis